apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector/preview.scss

```diff
@@ -0,0 +1,188 @@
+@import './shared.scss';
+
+// Preview positioning
+$preview-max-width: 960px;
+$preview-right-margin: 24px;
+$preview-left-sidebar-reduced: calc( 30% + #{$wp-org-sidebar-reduced} );
+$preview-left-sidebar-full: calc( 30% + #{$wp-org-sidebar-full} );
+
+$template-large-preview-title-height: 120px;
```

Comment:
I believe these will become global, there's no module-scoping of variables in SASS.

It would be good to make sure "local" variables have namespace style prefixes.

Reply:
I copied all of this from existing SPT styling :sweat_smile: Think there's potential for regression?